src/OrchardCore.Modules/OrchardCore.Resources/Assets.json

```diff
@@ -29,6 +29,12 @@
     "source": "../../../node_modules/bootstrap/dist/css/bootstrap.min.css",
     "tags": [ "resources", "css" ]
   },
+  {
+    "action": "copy",
+    "name": "bootstrap",
+    "source": "../../../node_modules/bootstrap/dist/css/bootstrap.min.css.map",
```

Comment:
@Skrypt if we can allow source to support array, it would clean up the Assets.json file. Alternately, you can add support for `sources` which supports array only.

Reply:
For copy I guess we could.